web/docs/advanced/links.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Type-Safe Links
+---
+
+import { Required } from '@site/src/components/Required'
+
+If you are using Typescript, you can use Wasp's custom `Link` component to create type-safe links to other pages on your site.
+
+## Using the `Link` Component
+
+After you defined a route:
+
+```wasp
+route TaskRoute { path: "/task/:id", to: TaskPage }
+page TaskPage { ... }
+```
+
+You can get the benefits of type-safe links by using the `Link` component from `@wasp/router`:
+
+```jsx
+import { Link } from '@wasp/router'
+
+export const TaskList = () => {
+  // ...
+
+  return (
+    <div>
+      {tasks.map((task) => (
+        <Link
+          key={task.id}
+          to="/task/:id"
+      {/* 👆 You must provide a valid path here */} 
+          params={{ id: task.id }}>
+      {/* 👆 All the params must be correctly passed in */}   
```

Comment:
I'd skip the pointer emojis, but to each their own :)

Reply:
If we plan on restricting emojis in docs in the new style-guide for docs, I'll remove it, np 👍

Reply:
I'll leave it for since we have them elsewhere, but I'll remove them if we start removing emojis from all parts of the docs.